app/assets/stylesheets/organizations.scss

```diff
@@ -9,11 +9,23 @@
 }
 
 .opp-info {
+  margin-top: 5%;
   text-align: center;
-  border: 1px solid #FF851B;
+  border: 2px solid #FF851B;
 }
 
 .org-button {
   background-color: #FF851B;
   color: white;
 }
+
+.opp-button {
+  background-color: #FF851B;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.